jeremy2918/tsseract
# Added missing Html tag

# [Added missing Html tag]


This PR adds the missing Html tag to eliminate the warning Nextjs implemented in the lasted update

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All test should be passing successfully.
- `npm run watch`
- Check that you do not have any warnings by nextjs.



## UI Changes
NO UI Changes
